src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+
+@Entity
```

Comment:
https://www.baeldung.com/hibernate-inheritance
Check what specific annotation we should use for a particular inheritance type